qdev_wrappers/fitting/Converter.py

```diff
@@ -0,0 +1,236 @@
+import qcodes as qc
+import numpy as np
+from os.path import sep
+from qcodes.dataset.data_export import get_data_by_id
+from qcodes.dataset.sqlite_base import connect
+from qdev_wrappers.file_setup import CURRENT_EXPERIMENT
+from qdev_wrappers.show_num import check_experiment_is_initialized
+
+
+def data_to_dict(id, samplefolder=None, datatype='SQL'):
+    if datatype == 'qcodes_legacy':
+        converter = Legacy_Converter()
+
+    if datatype == 'SQL':
+        converter = SQL_Converter()
+
+    return converter.convert(id, samplefolder)
+
+
+class DataConverter():
```

Comment:
It would be great if you could define the used functions and let them throw an exceptions to mark that they are not implemented in this base class. (This would be the classical oop way to do things. Possibly it is fine in Python not to do this, but then I would add a comment that this class is meant to be used as a base class for ... and ... or call it a mixin class.)

Reply:
also would be a fab place to describe what each function should do and then in the classes which inherit you don't need to do it :)